pyproject.toml

```diff
@@ -12,6 +12,6 @@ multi_line_output = 3
 include_trailing_comma = true
 force_grid_wrap = 0
 use_parentheses = true
-line_length = 88
+line_length = 80
```

Comment:
This is configured to 88 because that's the max limit Black allows when it is configured to try to keep to 80.

Reply:
This project has changed the default black line length, which requires this change in isort too.

https://github.com/mopidy/mopidy/blob/8954abcf9dcc8ff55755d2a858345a33d732a7c8/pyproject.toml#L7

Let me test `profile=black` add above and see if it uses the black config.

Reply:
I see what you are saying now, but black failed when using `line-length=88` in isort.